website/client/src/components/tasks/taskModal.vue

```diff
@@ -219,6 +219,23 @@
             @select="setDifficulty($event)"
           />
         </template>
+        <div
+          v-if="task.type === 'todo' && (!challengeAccessRequired || task.startShowing)"
```

Comment:
That way the input entry will be hidden if there is no value set & if we are in a challenge without write access.. so that might be a nice feature and keeps the UI clean if there is none